wagtail/admin/userbar.py

```diff
@@ -25,13 +26,47 @@ def render(self, request):
 class AccessibilityItem(BaseItem):
     template = "wagtailadmin/userbar/item_accessibility.html"
 
+    def get_axe_configuration(self):
```

Comment:
Not sure if it's worth passing the `request` object here but I opted not to, at least for now.